packages/xstate-wallet/src/chain.ts

```diff
@@ -347,7 +347,7 @@ export class ChainWatcher implements Chain {
 
     const response = await this.signer.sendTransaction({
       ...convertNitroTransactionRequest(transactionRequest),
-      gasPrice: GAS_PRICE
+      maxPriorityFeePerGas: GAS_PRICE
```

Comment:
Nit: Is this a good way to migrate the transaction type? Here's a good reference https://www.liquity.org/blog/how-eip-1559-changes-the-transaction-fees-of-ethereum . 

`GAS_PRICE` is defined on L244 with some reference to Goerli network. 

In the short term, we can do whatever seems to work. But we should have an eye on things like this which may break once deployed to Filecoin mainnet.

I think one alternative is to query the chain first to get a `maxPriorityFeePerGas`.